integration/src/test/java/org/mule/shutdown/ValidShutdownTimeoutRequestResponseTestCase.java

```diff
@@ -63,11 +65,15 @@ public void testExpressionTransformer() throws Throwable {
   private void doShutDownTest(final String payload, final String url) throws Throwable {
     final Future<?> requestTask = executor.submit(() -> {
       try {
-        HttpRequest request =
-            HttpRequest.builder().uri(url).method(GET).entity(new ByteArrayHttpEntity(payload.getBytes())).build();
-        final HttpResponse response =
-            httpClient.send(request, HttpRequestOptions.builder().responseTimeout(RECEIVE_TIMEOUT * 5).build());
-        assertThat("Was not able to process message ", IOUtils.toString(response.getEntity().getContent()), is(payload));
+        new PollingProber().check(new JUnitLambdaProbe(() -> {
+          HttpRequest request =
+              HttpRequest.builder().uri(url).method(GET).entity(new ByteArrayHttpEntity(payload.getBytes())).build();
+          final HttpResponse response =
+              httpClient.send(request, 5000, false, null);
```

Comment:
Why not use `send(HttpRequest, HttpRequestOptions)` instead of the deprecated one?

Reply:
Done!